src/complex.cr

```diff
@@ -251,6 +251,15 @@ struct Complex
     new 0, 0
   end
 
+  # Returns `true` if the complex number is zero.  This means the real and imaginary are both zero.
```

Comment:
```suggestion
  # Returns `true` if the complex number is zero, 
  # This means the real and imaginary are both zero.
```

Reply:
Comma?

Reply:
Newline, and yes, comma.

Reply:
That combination doesn't make sense to me. It's either comma *or* newline.

But these are really two separate sentences. Concatenating them does no good. So no comma, please. Newline is fine.